--- /Users/mjelen/Library/Application Support/Marked/Custom CSS/Custom Example.css
+++ (clipboard)
@@ -1,4 +1,4 @@
-/*
+/*
    This document has been created with Marked.app <http://markedapp.com>, Copyright 2011 Brett Terpstra
    Please leave this notice in place, along with any additional credits below.
    ---------------------------------------------------------------
@@ -241,7 +241,7 @@

 figcaption
 {
-    font-style:italic;
+    font-style:italic;
     text-align:center;
     background:rgba(0,0,0,.9);
     color:rgba(255,255,255,1);
@@ -373,6 +373,5 @@
     code span::selection {background-color:rgba(0,0,0,.7) !important;color:#eee !important}
     a::selection { background-color:rgba(255, 230, 102,.2)}
     .inverted a::selection { background-color:rgba(255, 230, 102,.6)}
- td::selection,th::selection,caption::selection { background-color: rgba(180, 237, 95, .5);}
-}
-
+  td::selection,th::selection,caption::selection { background-color: rgba(180, 237, 95, .5);}
+}
